core/src/utilities/helpers/navigation-helpers.js

```diff
@@ -141,13 +142,30 @@ class NavigationHelpersClass {
       if (!arr.metaInfo) {
         arr.metaInfo = metaInfo;
       }
+      if (!arr.metaInfo.collapsible && metaInfo.collapsible) {
+        arr.metaInfo.collapsible = metaInfo.collapsible;
+      }
+      if (category && GenericHelpers.isObject(category) && category.id && category.label) {
+        arr.metaInfo.label = category.label;
+        arr.metaInfo.id = category.id;
```

Comment:
```suggestion
        arr.metaInfo = { ...arr.metaInfo, { label: category.label, id: category.id } }
```
Use ES6 spread operator is nicer. ✌️

Reply:
@JohannesDoberer , adjusted the syntax of suggestion. Please check again. Thanks